### ISSUE TITLE
in demo case, when i press and release cmd + A,  key 'A' not restored.

### ISSUE THREAD

**Issue by 734vin** [ocornut/imgui#5128]
- problem 
in demo case, when i press and release cmd + A,  key 'A' not restored.  
this problem occurs with any combination key pressed.

- screenshot
<img width="1204" alt="2022-03-21 15 36 48" src="https://user-images.githubusercontent.com/56961542/159221289-4601820c-bbba-4074-8c22-a13d5c495c0e.png">

- Config/Build info

```
Dear ImGui 1.88 WIP (18709)
--------------------------------
sizeof(size_t): 8, sizeof(ImDrawIdx): 2, sizeof(ImDrawVert): 20
define: __cplusplus=201402
define: __APPLE__
define: __GNUC__=4
define: __clang_version__=13.0.0 (clang-1300.0.29.30)
define: IMGUI_HAS_VIEWPORT
define: IMGUI_HAS_DOCK
--------------------------------
io.BackendPlatformName: imgui_impl_osx
io.BackendRendererName: imgui_impl_metal
io.ConfigFlags: 0x00000000
io.ConfigViewportsNoDecoration
io.ConfigMacOSXBehaviors
io.ConfigInputTextCursorBlink
io.ConfigWindowsResizeFromEdges
io.ConfigMemoryCompactTimer = 60.0
io.BackendFlags: 0x0000000A
 HasMouseCursors
 RendererHasVtxOffset
--------------------------------
io.Fonts: 1 fonts, Flags: 0x00000000, TexSize: 512,64
io.DisplaySize: 1200.00,720.00
io.DisplayFramebufferScale: 2.00,2.00
--------------------------------
style.WindowPadding: 8.00,8.00
style.WindowBorderSize: 1.00
style.FramePadding: 4.00,3.00
style.FrameRounding: 0.00
style.FrameBorderSize: 0.00
style.ItemSpacing: 8.00,4.00
style.ItemInnerSpacing: 4.00,4.00
```

**Comment by ocornut** [comment]
It is your own app or it is the unmodified app from the examples/ folder ?

**Comment by 734vin** [comment]
i found in my app and try to locate it, but failed.
### then i compared prue example, it's reappeared

**Comment by 734vin** [comment]
> It is your own app or it is the unmodified app from the examples/ folder ?

i found in my app, i try to locate and fix it in my project code, but failed.
then 
i compared with `"pure example" in examples folder`, it's reappeared.

**Comment by ocornut** [comment]
Thanks for reporting!
Pushed fix 3c07879504adec0d1b96c10c0f1578c009e070ae by @thedmd
